src/csharp/Grpc.Core/SourceLink.csproj.include

```diff
@@ -0,0 +1,19 @@
+<!-- Ensure that debugging of the resulting NuGet packages work (we're using SourceLink). -->
+<Project>
+
+  <ItemGroup Label="dotnet pack instructions">
+    <Content Include="$(OutputPath)netstandard1.5\$(PackageId).pdb">
```

Comment:
@jskeet   i'm not sure if this should be `Content`  or `None`.  AFAIR, `Content` has some side effects on older .NET projects (like the files showing up in Visual Studio project explorer and being copied to the output directory for projects that include our packages), which I'm not sure is desirable here. Any opinions?

Reply:
We use Content. Yes, it shows up in project explorer (when developing the package itself), which isn't ideal - but it does seem to work, which is probably more important.
I'm hoping that "reasonably soon" all of this will be handled transparently. This follows the suggested format though.